ibm/service/power/resource_ibm_pi_instance.go

```diff
@@ -259,16 +259,15 @@ func ResourceIBMPIInstance() *schema.Resource {
 			PISAPInstanceProfileID: {
 				Type:          schema.TypeString,
 				Optional:      true,
-				ConflictsWith: []string{helpers.PIInstanceProcessors, helpers.PIInstanceMemory, helpers.PIInstanceProcType, helpers.PIInstanceSystemType},
+				ConflictsWith: []string{helpers.PIInstanceProcessors, helpers.PIInstanceMemory, helpers.PIInstanceProcType},
 				Description:   "SAP Profile ID for the amount of cores and memory",
 			},
 			helpers.PIInstanceSystemType: {
-				Type:          schema.TypeString,
-				Optional:      true,
-				Computed:      true,
-				ValidateFunc:  validate.ValidateAllowedStringValues([]string{"s922", "e880", "e980"}),
-				ConflictsWith: []string{PISAPInstanceProfileID},
-				Description:   "PI Instance system type",
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				ValidateFunc: validate.ValidateAllowedStringValues([]string{"s922", "e880", "e980", "e1080"}),
```

Comment:
IMO we should remove the validate check as more system types are allowed for pvm instance such as `s1022` which is not possible to use via Terraform today.
You could have a dynamic check specific for SAP create func to allow selected system types.

Reply:
I'll just remove the validate check for this and we can let the service broker check for the correct SAP sysType